src/DGtal/dec/DiscreteExteriorCalculus.ih

```diff
@@ -270,6 +270,140 @@ DGtal::DiscreteExteriorCalculus<dimEmbedded, dimAmbient, TLinearAlgebraBackend,
   return _operator;
 }
 
+template <DGtal::Dimension dimEmbedded, DGtal::Dimension dimAmbient, typename TLinearAlgebraBackend, typename TInteger>
+template <DGtal::Duality duality, typename TPointMapper>
+DGtal::LinearOperator<DGtal::DiscreteExteriorCalculus<dimEmbedded, dimAmbient, TLinearAlgebraBackend, TInteger>, 0, duality, 0, duality>
+DGtal::DiscreteExteriorCalculus<dimEmbedded, dimAmbient, TLinearAlgebraBackend, TInteger>::quadLaplace(const Scalar& h, const Scalar& lambda,
```

Comment:
Actually if you look down into the implementation of the discrete calculus, methods are quite long in general...

Reply:
I see, but could not that be split up? Are there any reasons for keeping this that long? It seems like there are many things going on. I would say, divide and conquer ;-)

Reply:
`template <DGtal::Duality duality, typename TPointMapper>` is actually the only different part from other methods (for example the laplace() one). As for splitting the method, it would en in a loss of performance because we're doing a single loop to compute all the relevent matrices for the laplace operator and it would also not decrease the size of the prototype ;)

Reply:
Well, the performance argument seems to convince me. Let it be ;-)